plugin/admin/src/components/BatchUpdateTable/Table.js

```diff
@@ -0,0 +1,230 @@
+  const { formatMessage } = useIntl()
+
+  const [selectedIDs, setSelectedIDs] = useState([])
+  const [dialogOpen, setDialogOpen] = useState(false)
+  const [isLoading, setIsLoading] = useState(false)
+  const [sourceLocale, setSourceLocale] = useState(null)
+
+  const { locales } = useCollection()
+
+  useEffect(() => {
+    if (Array.isArray(locales) && locales.length > 0)
+      setSourceLocale(locales[0].code)
+  }, [locales])
+
+  const { updates, startUpdate, dismissUpdates, refetch } =
+    useUpdateCollection()
+
+  useEffect(() => {
+    const localeUses = updates
+      .filter(({ id }) => selectedIDs.includes(id))
+      .reduce((acc, current) => {
+        for (const locale of current.attributes.localesWithUpdates) {
+          acc[locale] = (acc[locale] ?? 0) + 1
+        }
+
+        return acc
+      }, {})
+
+    const { code: mostUpdatedLocale } = Object.entries(localeUses).reduce(
+      (acc, [code, uses]) => (acc.uses < uses ? { code, uses } : acc),
+      { code: '', uses: 0 }
+    )
+
+    setSourceLocale(mostUpdatedLocale)
+  }, [selectedIDs, updates])
+
+  const onAction = () => {
+    setIsLoading(true)
+    startUpdate(selectedIDs, sourceLocale).then(() => {
+      setIsLoading(false)
+      setDialogOpen(false)
+      refetch()
+    })
+  }
+
+  const dismissSelected = () => {
+    dismissUpdates(selectedIDs).then(() => {
+      setSelectedIDs([])
+      refetch()
+    })
+  }
+
+  return (
+    <Box background="neutral100">
+      <Flex>
+        <Button
+          disabled={selectedIDs.length === 0}
+          onClick={() => setDialogOpen(true)}
+        >
+          Translate selected
+        </Button>
+        <Button
+          disabled={selectedIDs.length === 0}
+          onClick={() => dismissSelected()}
+          variant="danger"
+        >
+          Dismiss selected
+        </Button>
+      </Flex>
+      <Table>
+        <Thead>
+          <Tr>
+            <Th>
+              <Checkbox
+                value={selectedIDs.length > 0}
+                disabled={updates.length === 0}
+                indeterminate={
+                  selectedIDs.length > 0 && selectedIDs.length < updates.length
+                }
+                onClick={() => {
+                  if (selectedIDs.length === updates.length) {
+                    setSelectedIDs([])
+                  } else {
+                    setSelectedIDs(updates.map(({ id }) => id))
+                  }
+                }}
+              />
+            </Th>
+            <Th>
+              <Typography variant="sigma">Type</Typography>
+            </Th>
+            <Th>
+              <Typography variant="sigma">IDs</Typography>
+            </Th>
+            {locales.map((locale) => (
+              <Th key={locale.code}>
+                <Typography variant="sigma">{locale.name}</Typography>
+              </Th>
+            ))}
+          </Tr>
+        </Thead>
+        <Tbody>
+          {updates.map(({ id, attributes }, index) => (
```

Comment:
I feel like this table becomes unreadable when many entites are updated. Would there be a possiblity to group these updates by content type?

Optimally I would even like to have only One table, so would it be possible to integrate the functionality into the existing table?

Reply:
I think the way it is implemented atm, it should be in a separate table. Unlike the batch translations, which works on a content type level (translating every entity of a given content type from one source locale to one target locale), batch updating works on an entity level (all translations of a given entity are retranslated from a given source locale). 

By default, there is now only one row per content type, which can be expanded to show all entries.